.github/workflows/bump-auxiliary-packages.yml

```diff
@@ -41,8 +49,6 @@ jobs:
       - name: Bump packages
         run: |
           npm run bump-auxiliary
-          git add .
-          git commit --no-allow-empty -m "chore(release): bump packages for auxiliary release" || true
```

Comment:
peter-evans/create-pull-request already commits all changes, so there's no reason for us to manually do it.